### PR TITLE
add ability to theme song text independently

### PR DESCRIPTION
**Pull request by finity69x2**
As asked for in the release thread. I figured I'd just do it to help you out.

Hopefully it is OK.

Also, I just realized I forgot to update the docs.

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -207,6 +207,7 @@ export class CustomSonosCard extends LitElement {
           --sonos-int-player-section-background: var(--sonos-player-section-background, #ffffffe6);
           --sonos-int-color: var(--sonos-color, var(--secondary-text-color));
           --sonos-int-artist-album-text-color: var(--sonos-artist-album-text-color, var(--secondary-text-color));
+          --sonos-int-song-text-color: var(--sonos-song-text-color, var(--accent-color));
```
Comment (johanfrick) [inline src/main.ts:210]: This should default back to Sonos accent Color.
Also please update readme 🙂

**Review by johanfrick — CHANGES_REQUESTED** [review]

**Comment by johanfrick** [comment]
Thanks for helping out! Just some minor tweaks needed 🙂

**Review by johanfrick — APPROVED** [review]
👍👍🍾

**Comment by finity69x2** [comment]
I hope that is what you meant. I'm not sure how variables are inherited.

updated readme as well.

**Comment by johanfrick** [comment]
Really nice @finity69x2 !
I’ll make a release tomorrow 🙂

**Comment by johanfrick** [comment]
https://github.com/johanfrick/custom-sonos-card/releases/tag/v4.1.0

**Pull request merged**